### PR TITLE
D-Bus fix

### PR DESCRIPTION
**Pull request by divyk**
### Change Description ###

I raised this PR for changing the usage of dbus to D-Bus, to maintain consistency. Please review.

### CODE REVIEW

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 1]
```diff
@@ -990,7 +990,7 @@ New Features
 - Camera 
   - Flash control behavior has changed. (Flash control can fail if the flash is pre-empted by another API.)
   - Code for removing remaining messages when destroying a handle has been updated.
-  - dbus has been replaced with gdbus.
+  - D-Bus has been replaced with gdbus.
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-2-4.md]: gdbus -> GDBus

File: docs/open-source-project/release-notes/tizen-3-0-m2.md [hunk 2]
```diff
@@ -535,7 +535,7 @@ As a common feature, multi-user support has been added.
 - Major open-source component upgrades and changes
   - Curl has been upgraded from 7.40 to 7.50.2 for stability.
   - Dnsmasq has been upgraded from 2.57 to 2.74 for stability.
-  - wpa_supplicant has been upgraded from 2.4 to 2.5 for the d-bus interface.
+  - wpa_supplicant has been upgraded from 2.4 to 2.5 for the D-Bus interface.
   - libidn has been upgraded from 1.15 to 1.16 for stability.
   - Libcap has been upgraded from 2.21 to 2.24 for stability.
   - Gnutls has been upgraded from 3.3.5 to 3.4.11 for stability.
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-3-0-m2.md]: On line 595, we have "gdbus" -> GDBus

On line 745, we have "gdbus" -> GDBus

File: docs/open-source-project/release-notes/tizen-3-0-m2.md [hunk 3]
```diff
@@ -247,7 +247,7 @@ As a common feature, multi-user support has been added.
     - EGL&trade; 1.4
     - OpenGL&reg; ES 1.1, 2.0, 3.0, 3.1, 3.2
   - OpenGL&reg; ES/EGL&trade; development headers are provided.
-    - Khronos OpenGL&reg; ES/EGL&trade; headers (Khronos released on Oct 24 2016)
+    - Khronos OpenGL&reg; ES/EGL&trade; headers (Khronos released on Oct 24 2016).
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-3-0-m2.md]: Is this period really needed? Or was it to be inside of parenthesis?

File: docs/open-source-project/release-notes/tizen-3-0-m2.md [hunk 4]
```diff
@@ -662,38 +662,38 @@ As a common feature, multi-user support has been added.
...
+  - Yet Another Crypto API (YACA)
     - A new crypto API package, YACA, has been introduced in Tizen 3.0.
-      - It provides the application with the following stable cryptographic APIs:
-        - APIs for encryption/decryption operations with symmetric keys and sealing and opening operations with asymmetric keys
-        - APIs for creating and verifying a signature, calculating HMAC/CMAC, and calculating a message digest
-        - APIs for key handling operations, such as generating, importing, and exporting a key, and deriving a key from a password
-        - Simple APIs for cryptographic operations
-        - APIs for low-level RSA operations
+      - It provides the application with the following stable cryptographic API set:
+        - API set for encryption/decryption operations with symmetric keys and sealing and opening operations with asymmetric keys.
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-3-0-m2.md]: From line 684 to line 688, it looks like they are not "sentences".  If so, are they need period?

File: docs/open-source-project/release-notes/tizen-4-0-m1.md [hunk 5]
```diff
@@ -154,7 +154,7 @@ The Tizen 4.0 Public M1 release provides developers with the Tizen kernel, devic
 - Tizen HAL
   - Tizen display HAL
     - Support has been added for the layer commit to ensure an output commit per each vblank:
-      - `TDM_COMMIT_PER_VBLANK` environment
+      - `TDM_COMMIT_PER_VBLANK` environment.
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-4-0-m1.md]: It looks like this line is not a "sentence". If so, does this line need period?

File: docs/open-source-project/release-notes/tizen-4-0-m2.md [hunk 6]
```diff
@@ -451,11 +451,11 @@ The Tizen 4.0 Public M2 release provides developers with the Tizen kernel, devic
     - iptables has been upgraded from 1.4.21 to 1.6.1.
     - openvpn has been upgraded from 2.3.2 to 2.4.2.
 - Bluetooth
-  - New features and APIs:
-    - The C# Public OPP Client API has been added.
-    - The C# Public OPP Server API has been added.
+  - New features and API set:
+    - The C# OPP Client API has been added.
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-4-0-m2.md]: From line 455 to line 4556 : Why "Public" is removed?

File: docs/open-source-project/release-notes/tizen-4-0-m2.md [hunk 7]
```diff
@@ -26,10 +26,10 @@ The Tizen 4.0 Public M2 release provides developers with the Tizen kernel, devic
 - System Framework
   - Extended internal storage support, without on-demand device encryption, has been added:
     - Extended internal storage is an SD card-based storage extension feature.
-  - Dbus debugging and profiling tools (eBPF-based monitoring) have been added.
-  - Supplementary group support for libdbuspolicy with Kdbus backend has been added.
-  - C# Public APIs (device, feedback, storage, dlog, tizen-platform-config (internal)) has been added.
-  - New Public enumerators have been added to libsvi, such as FEEDBACK_PATTERN_SYSTEM_SHORT.
+  - D-Bus debugging and profiling tools (eBPF-based monitoring) have been added.
+  - Supplementary group support for libdbuspolicy with KDBus backend has been added.
+  - C# API set (device, feedback, storage, dlog, tizen-platform-config (internal)) has been added.
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-4-0-m2.md]: Why "Public" in this line and line 32 are removed?

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 8]
```diff
@@ -2,7 +2,7 @@
 
 Release Date: Oct 30, 2015
 
-The Tizen 2.4 release provides developers with the Tizen kernel, device drivers, middleware subsystems, and Web/native APIs, necessary to develop future Tizen compliant solutions.
+The Tizen 2.4 release provides developers with the Tizen kernel, device drivers, middleware subsystems, and Web/native API set, necessary to develop future Tizen compliant solutions.
 
 #### Release Details
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 9]
```diff
@@ -26,7 +26,9 @@ The Tizen 2.4 release provides developers with the Tizen kernel, device drivers,
 
 - Native UI Builder
   - When you build the UI of a wearable application, you cannot use the Panel UI component. The component is not supported in wearable applications.
-  - Support for new APIs for the wearable widget have been added. Old APIs in the project code of the wearable widget, which were shared with the wearable application, have been replaced with the new APIs.
+  - Support for new The Tizen 2.4 release provides developers with the Tizen kernel, device drivers, middleware subsystems, and Web/native API set, necessary to develop future Tizen compliant solutions.
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Have re-framed the sentence , please review

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 10]
```diff
@@ -36,7 +38,7 @@ The Tizen 2.4 release provides developers with the Tizen kernel, device drivers,
   - When you tried to debug an application running on a connected target device through the **Debug As > Tizen Native Application – Attach **menu, you could not attach the debugger to the application in the **Debug Configurations** dialog. This bug has been fixed.
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 11]
```diff
@@ -310,7 +312,7 @@ The Tizen 2.4 release provides developers with the Tizen kernel, device drivers,
 
 
 - Native UI Builder
-  - The **Resource View** has been replaced by the **Resource Manager View**When you open an older project version than 2.4 Rev5, the **Resource View** still remains in the perspective. To update to the **Resource Manager View**, click **Tizen Native** on the Perspective Toolbar, open the context menu, and click **Reset**.
+  - The **Resource View** has been replaced by the **Resource Manager View**. When you open an earlier version than 2.4 Rev5, the **Resource View** still remains in the perspective. To update to the **Resource Manager View**, click **Tizen Native** on the Perspective Toolbar, open the context menu, and click **Reset**.
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Accept

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 12]
```diff
@@ -383,8 +385,8 @@ The Tizen 2.4 release provides developers with the Tizen kernel, device drivers,
...
   - On Ubuntu 14.04, a shortcut menu can sometimes appear transparent.
   - On Windows, depending on your OS theme (such as Non-Aero themes and Windows XP themes), a display surface can be erased for a while if the Emulator window is covered with another window. If you click the Emulator window, the display surface runs correctly again
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 13]
```diff
@@ -472,7 +474,7 @@ The Tizen 2.4 release provides developers with the Tizen kernel, device drivers,
 **Known Issues**
 
 - Installer and Update Manager
-  -  The update command of the 2.4 rev1 and rev2 CLI installer has some errors. If you use the 2.4 rev1 or rev2 CLI installer, the Tizen SDK update fails.  To solve the problem, download [CLI installer patch files](https://download.tizen.org/sdk/patches/), and do the following (or upgrade your CLI installer to the 2.4 rev3 at the [Tizen SDK Download page](https://developer.tizen.org/development/tools/download).):
+  -  The update command of the 2.4 rev1 and rev2 CLI installer has some errors. If you use the 2.4 rev1 or rev2 CLI installer, the Tizen SDK update fails.  To solve the problem, download [CLI installer patch files](https://download.tizen.org/sdk/patches/), and do the following (or upgrade your CLI installer to the 2.4 rev3 at the [Tizen SDK Download page](https://developer.tizen.org/development/tools/download):
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 14]
```diff
@@ -719,7 +721,7 @@ The Tizen 2.4 release provides developers with the Tizen kernel, device drivers,
   - The content/code assist feature has been added for the **i18n_get_text()** function. After you complete the **i18n_get_text()** function on the C/C++ source file editor by pressing **Ctrl + Space**, you can see a candidate parameter list which consists of original untranslated strings you added to the PO file.
   - When you develop a multi-project packaged application (such as a combination of UI and service projects, or UI and shared library projects), the res.xml files of the projects are automatically merged. If a project has no res.xml file, a res.xml is automatically generated for that project and merged with the other res.xml files. 
   - The “Export to CLI” menu has been added, and appears when multiple projects are selected, .
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 15]
```diff
@@ -728,22 +730,23 @@ The Tizen 2.4 release provides developers with the Tizen kernel, device drivers,
...
 
 - Installer and Update Manager
   - If there is a multibyte character in the installation path of the Tizen SDK, some development packages cannot find the installed SDK’s location when they are working.
-  - When you install packages by using the All Packages tab in the Update Manager, the number of the progress indicator is only changed each time a package in dependency is downloaded. While downloading, the number of the progress indicator is not changed. 
+  - When you install packages by using the **All Packages** tab in the Update Manager, the number of the progress indicator is only changed each time a package in dependency is downloaded. While downloading, the number of the progress indicator is not changed. 
   - When you run the dynamic analyzer on Mac OS, the SDK Update Notification does not appear though there are available SDK updates.
-  - If the SDK Update Notification appears when you start the Tizen IDE or Emulator on Mac OS, a terminal (shell.exec) icon can appear on the dock for a few seconds.
+  - If the *SDK Update Notification* appears when you start the Tizen IDE or Emulator on Mac OS, a terminal (shell.exec) icon can appear on the dock for a few seconds.
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 16]
```diff
@@ -880,44 +883,44 @@ New Features
 - Open source upgrade
   - EFL version has been upgraded from 1.7 to 1.13.
 - API changes
-  - Eina_Bool edje_text_class_get (const char *text_class, char **font, Evas_Font_Size *size) has been changed to  Eina_Bool edje_text_class_get (const char *text_class, const char **font, Evas_Font_Size *size).
-  - void elm_win_wm_rotation_preferred_rotation_set(Evas_Object *obj, const int rotation) has been changed to void elm_win_wm_rotation_preferred_rotation_set(const Evas_Object *obj, int rotation).
+  - Eina_Bool edje_text_class_get (const char *text_class, char* font, *Evas_Font_Size* size) has been changed to  Eina_Bool edje_text_class_get (const char *text_class, const char* font, *Evas_Font_Size* size).
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 17]
```diff
@@ -880,44 +883,44 @@ New Features
...
-  - New APIs for a call status have been added.
-  - New APIs for a modem power status have been added.
-  - New APIs for SIM application lists have been added.
-  - New APIs for a network have been added.
+  - New API set for a call status have been added.
+  - New API set for a modem power status have been added.
+  - New API set for SIM application lists have been added.
+  - New API set for a network have been added.
 
 ** Change Notes**
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 18]
```diff
@@ -1014,22 +1017,22 @@ New Features
...
+    - Account connect/disconnect API set have been deprecated to avoid applications using them inappropriately.
   - Sync Manager
-    - New APIs to schedule the operation of applications which need synchronization with a server have been added.
+    - New API set to schedule the operation of applications which need synchronization with a server have been added.
   - OAuth 2.0
-    -  New APIs for easy usage of the OAuth 2.0 protocol have been added.
+    -  New API set for easy usage of the OAuth 2.0 protocol have been added.
 - Location FW 
   - Location Manager
     - New API for distance-based location updates has been added.
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 19]
```diff
@@ -1164,7 +1167,7 @@ New Features
   - Support for DRI3 and Present extension has been added.
   - Support for HWA (hardware access) extension has been added.
 -  EOM
-  - New APIs for EOM (External Output Manager) have been added.
+  - New API set for External Output Manager (EOM) have been added.
 -  Window Manager
   - Support for window and input transformation has been added.
   - Support for starting a window with an iconic state has been added.
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 20]
```diff
@@ -1264,7 +1267,7 @@ New Features
...
+- BT privileges ([http://tizen.org/privilege/bluetooth.xx](http://tizen.org/privilege/bluetooth.xx)) have been deprecated to synchronize the privilege names between Web and native Bluetooth API set. From 2.4, [http://tizen.org/privilege/bluetooth](http://tizen.org/privilege/bluetooth) must be used instead of the listed [http://tizen.org/privilege/bluetooth.xx](http://tizen.org/privilege/bluetooth.xx) privileges.[http://tizen.org/privilege/bluetooth.xx](http://tizen.org/privilege/bluetooth.xx) -> [http://tizen.org/privilege/bluetooth](http://tizen.org/privilege/bluetooth)[http://tizen.org/privilege/bluetooth.admin](http://tizen.org/privilege/bluetooth.admin)[http://tizen.org/privilege/bluetooth.gap](http://tizen.org/privilege/bluetooth.gap)[http://tizen.org/privilege/bluetooth.health](http://tizen.org/privilege/bluetooth.health)[http://tizen.org/privilege/bluetooth.spp](http://tizen.org/privilege/bluetooth.spp)
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 21]
```diff
@@ -1430,7 +1433,7 @@ New Features
 **Sync Manager**
 
 - For API enhancement, several changes have been applied to Sync Manager. For more detailed information, see the API Reference.
-- [Changed] 2 APIs may require a proper privilege to work correctly:
+- [Changed] 2 API set may require a proper privilege to work correctly:
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 22]
```diff
@@ -1509,7 +1512,7 @@ New Features
 
 **Application Framework**
 
-- [Deprecated] 2 APIs that are not supported since 2.4 have been deprecated:
+- [Deprecated] 2 API set that are not supported since 2.4 have been deprecated:
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-3-0-m2.md (hunk 2 above)
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-3-0-m2.md]: I can see that I have already made the changes, I even checked the output to confirm.

File: docs/open-source-project/release-notes/tizen-3-0-m2.md (hunk 4 above)
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-3-0-m2.md]: I have removed

File: docs/open-source-project/release-notes/tizen-4-0-m1.md (hunk 5 above)
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-4-0-m1.md]: I have removed the period

File: docs/open-source-project/release-notes/tizen-4-0-m2.md (hunk 7 above)
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-4-0-m2.md]: added

File: docs/open-source-project/release-notes/tizen-4-0-m2.md [hunk 23]
```diff
@@ -157,15 +157,15 @@ The Tizen 4.0 Public M2 release provides developers with the Tizen kernel, devic
...
   - A new layer has been added to support the cursor.
   - The transient_below feature has been added to support a client window which needs to be placed under the parent window.
   - The `sd_notify()` function call has been added to the main function for sending systemd start-up notifications, since the `ecore_main_loop_begin()` function in EFL no longer calls it.
   - Creating the keymap cache file is now processed before initializing the DRM.
   - A remote surface client check has been added to permit access only to processes with a privileged UID.
-  - A dbus policy check has been added to permit access only to processes with a privileged UID.
+  - A D-Bus policy check has been added to permit access only to processes with a privileged UID.
   - Auxiliary hints have been added for:
     - wm.policy.win.deiconify.update: to support clients which need to defer deiconify rendering.
     - wm.policy.win.iconify.buffer.flush: to support clients which need to change the buffer flush feature use when iconified.
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-4-0-m2.md]: Done

File: docs/open-source-project/release-notes/tizen-4-0-m2.md (hunk 6 above)
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-4-0-m2.md]: Added

File: docs/open-source-project/release-notes/tizen-4-0-m2.md [hunk 24]
```diff
@@ -26,9 +26,9 @@ The Tizen 4.0 Public M2 release provides developers with the Tizen kernel, devic
 - System Framework
   - Extended internal storage support, without on-demand device encryption, has been added:
     - Extended internal storage is an SD card-based storage extension feature.
-  - Dbus debugging and profiling tools (eBPF-based monitoring) have been added.
-  - Supplementary group support for libdbuspolicy with Kdbus backend has been added.
-  - C# Public APIs (device, feedback, storage, dlog, tizen-platform-config (internal)) has been added.
+  - D-Bus debugging and profiling tools (eBPF-based monitoring) have been added.
+  - Supplementary group support for libdbuspolicy with KDBus backend has been added.
+  - C# PublicAPI set (device, feedback, storage, dlog, tizen-platform-config (internal)) has been added.
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-4-0-m2.md]: Public API. A space is needed.

File: docs/open-source-project/release-notes/tizen-4-0-m1.md [hunk 25]
```diff
@@ -23,22 +23,22 @@ The Tizen 4.0 Public M1 release provides developers with the Tizen kernel, devic
...
     - Libgudev package has been detached from Systemd.
     - Initrd package has been added.
     - Libconfig has been upgraded to 1.6.
     - Libusb has been upgraded to 1.0.21.
   - The resource management daemon has been changed to make resources lighter for IoT devices.
-  - APIs have been added:
-    - Dlog C#-internal APIs
-    - Getting physical memory size API
+  - API set have been added:
+    - Dlog C#- API set.
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-4-0-m1.md]: Why "internal" is removed?

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 26]
```diff
@@ -880,44 +883,44 @@ New Features
 - Open source upgrade
   - EFL version has been upgraded from 1.7 to 1.13.
 - API changes
-  - Eina_Bool edje_text_class_get (const char *text_class, char **font, Evas_Font_Size *size) has been changed to  Eina_Bool edje_text_class_get (const char *text_class, const char **font, Evas_Font_Size *size).
-  - void elm_win_wm_rotation_preferred_rotation_set(Evas_Object *obj, const int rotation) has been changed to void elm_win_wm_rotation_preferred_rotation_set(const Evas_Object *obj, int rotation).
+  - Eina_Bool edje_text_class_get (**const char text_class**, **char font**, **Evas_Font_Size** size) has been changed to  Eina_Bool edje_text_class_get (**const char text_class**, **const char font**, **Evas_Font_Size** size).
+  - void elm_win_wm_rotation_preferred_rotation_set (Evas_Object *obj, const int rotation) has been changed to void elm_win_wm_rotation_preferred_rotation_set (const Evas_Object *obj, int rotation).
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-2-4.md]: Just copy below line and paste ate 887.

```
void elm_win_wm_rotation_preferred_rotation_set (Evas_Object \*obj, const int rotation) has been changed to void elm_win_wm_rotation_preferred_rotation_set (const Evas_Object \*obj, int rotation).
```

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 27]
```diff
@@ -880,44 +883,44 @@ New Features
 - Open source upgrade
   - EFL version has been upgraded from 1.7 to 1.13.
 - API changes
-  - Eina_Bool edje_text_class_get (const char *text_class, char **font, Evas_Font_Size *size) has been changed to  Eina_Bool edje_text_class_get (const char *text_class, const char **font, Evas_Font_Size *size).
-  - void elm_win_wm_rotation_preferred_rotation_set(Evas_Object *obj, const int rotation) has been changed to void elm_win_wm_rotation_preferred_rotation_set(const Evas_Object *obj, int rotation).
+  - Eina_Bool edje_text_class_get (**const char text_class**, **char font**, **Evas_Font_Size** size) has been changed to  Eina_Bool edje_text_class_get (**const char text_class**, **const char font**, **Evas_Font_Size** size).
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-2-4.md]: Just copy below line and paste at line 886.
```
Eina_Bool edje_text_class_get (const char \*text_class, char \*\*font, Evas_Font_Size \*size) has been changed to Eina_Bool edje_text_class_get (const char \*text_class, const char \*\*font, Evas_Font_Size \*size).
```

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 28]
```diff
@@ -26,17 +26,19 @@ The Tizen 2.4 release provides developers with the Tizen kernel, device drivers,
 
 - Native UI Builder
   - When you build the UI of a wearable application, you cannot use the Panel UI component. The component is not supported in wearable applications.
-  - Support for new APIs for the wearable widget have been added. Old APIs in the project code of the wearable widget, which were shared with the wearable application, have been replaced with the new APIs.
+  - Support for new API set for the wearbale widget have been added. Old API set in the project code of the wearble widget, which were shared with the werable application, have been replaced with the new API set.
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-2-4.md]: wearble -> wearable

werable -> wearable

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 29]
```diff
@@ -26,17 +26,19 @@ The Tizen 2.4 release provides developers with the Tizen kernel, device drivers,
...
+ - Support for the wearable widget have been added. The Tizen 2.4 release provides developers with the Tizen kernel, device drivers, middleware subsystems, Web API set, and Native API set necessary to develop future Tizen compliant solutions.
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-2-4.md]: Delete this line 30. There is no reason to add this sentence.

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 30]
```diff
@@ -26,17 +26,19 @@ The Tizen 2.4 release provides developers with the Tizen kernel, device drivers,
...
+ - Replaced new API set in the project code of the wearable widget, which were shared with the wearable application.
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-2-4.md]: Delete this line 31. There is no reason to add this sentence.

File: docs/open-source-project/release-notes/tizen-2-4.md (hunk 27 above)
Comment (divyk) [reply docs/open-source-project/release-notes/tizen-2-4.md]: Done

File: docs/open-source-project/release-notes/tizen-2-4.md [hunk 31]
```diff
@@ -26,17 +26,17 @@ The Tizen 2.4 release provides developers with the Tizen kernel, device drivers,
 
 - Native UI Builder
   - When you build the UI of a wearable application, you cannot use the Panel UI component. The component is not supported in wearable applications.
-  - Support for new APIs for the wearable widget have been added. Old APIs in the project code of the wearable widget, which were shared with the wearable application, have been replaced with the new APIs.
+  - Support for new API set for the wearabale widget have been added. Old API set in the project code of the wearble widget, which were shared with the werable application, have been replaced with the new API set.
```
Comment (mijin85cho) [inline docs/open-source-project/release-notes/tizen-2-4.md]: Fix "wearabale", "wearble" and "werable". Please stop increasing mis-typo.

**Comment by mijin80lee** [comment]
TD Sync Completed and other diffs will be fixed in addition.
And please check the following. 

docs/open-source-project/release-notes/tizen-2-4.md
line261: seperated sentences are merged in one. 
line313: seperated sentences are merged in one. 
line371: seperated sentences are merged in one. 
line372: seperated sentences are merged in one. 
line387: seperated sentences are merged in one. 
line476,477,478: indent one level.
line 685: remove two spaces.
line750: seperated sentences are merged in one. 
line760: seperated sentences are merged in one. 
line877: indent one level.
line1158,1159,1160,1161,1162,1163,1164,1165,1166: indent one level.
line1168: indent one level.
line1170,1171,1172,1173,1174: indent one level.

docs/open-source-project/release-notes/tizen-4-0-m1.md
line3: release date check(mm-dd-yy)or(dd-mm-yy)??

docs/open-source-project/release-notes/tizen-4-0-m2.md
line3: release date check(mm-dd-yy)or(dd-mm-yy)??
line44: change on-demand to On-demand??

**Pull request merged**